frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -40,12 +40,9 @@ export default function ViewFacility() {
     }
   }, [])
 
-  const MyBookingButton = (
-    <Button
-      state="primary"
-      text="Bookings"
-      type="button"
-      size="small"
+  const myBookingsIcon = (
```

Comment:
I feel that it is better to maintain naming scheme
```suggestion
  const MyBookingsIcon = (
```

Reply:
Fixed in #[5fbad19](https://github.com/rhdevs/rhapp/pull/639/commits/5fbad19aa74a40cdc7965b02d74c8f8d0498242a)